intelmq/bots/collectors/opendxl/collector.py

```diff
@@ -0,0 +1,106 @@
+"""
+openDXL Collector Bot
+Connects to a openDXL fabric and subscribes to ATD topic
+TLS is used by default.
+
+Parameters:
+dxl_config_file: string
+dxl_topic: string
+"""
+
+import logging
+import os
+import sys
+import time
+import json
+import threading
+import importlib
+
+from dxlclient.callbacks import EventCallback
```

Comment:
Please put this in a try-except block and do error handling in init, see for example https://github.com/certtools/intelmq/blob/maintenance/intelmq/bots/outputs/mongodb/output.py#L8